src/SteamAPI.ts

```diff
@@ -695,8 +695,8 @@ export default class SteamAPI {
 	 * Although a filter is not stricly required, you probably want to at least use something like \appid\[appid] to filter by app
 	 * @param count Number of results to return. 100 by default
```

Comment:
Change the docstring here to limit as well

Reply:
Done!